jack-sys/src/lib.rs

```diff
@@ -1,8 +1,14 @@
-#![allow(non_camel_case_types, non_upper_case_globals)]
+#![allow(non_camel_case_types, non_upper_case_globals, non_snake_case)]
 
 use dlib::external_library;
 use lazy_static::lazy_static;
 
+#[cfg(windows)]
+pub const JACK_LIB: &'static str = "libjack.dll";
```

Comment:
```suggestion
#[cfg(all(target_os = "windows", target_arch = "x86"))]
pub const JACK_LIB: &'static str = "libjack.dll";

#[cfg(all(target_os = "windows", any(target_arch = "x86_64", target_arch = "aarch64"))]
pub const JACK_LIB: &'static str = "libjack.dll";
```

https://github.com/jackaudio/jack2/issues/787#issuecomment-893782417

Reply:
Note that JACK builds don't actually support Windows ARM (currently) because MinGW does not support Windows ARM.

Reply:
This wasn't resolved? It's still using the same DLL name for both 32 and 64 bit Windows.